vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -107,29 +107,11 @@ def update_request(request, values, requester_id)
   def continue_request(values, _requester_id)
     return false unless validate(values)
 
-    if @running_pre_dialog == true
-      @values[:pre_dialog_vm_tags] = @values[:vm_tags].dup
-    end
-
-    @dialogs = get_dialogs
-
-    if @running_pre_dialog == true
-      @running_pre_dialog = false
-
-      if @values[:sysprep_domain_name].kind_of?(String) && !@values[:sysprep_domain_name].blank?
-        @values[:sysprep_domain_name] = [@values[:sysprep_domain_name], @values[:sysprep_domain_name]]
-        @values[:forced_sysprep_domain_name] = [@values[:sysprep_domain_name].first]
-      end
-
-      if @values[:sysprep_enabled].kind_of?(String) && !@values[:sysprep_enabled].blank?
-        @values[:forced_sysprep_enabled] = 'fields' if @values[:sysprep_enabled] == 'fields'
-      end
-    end
-
+    exit_pre_dialog if @running_pre_dialog
     password_helper(@values, false) # Decrypt passwords in the hash for the UI
+    @dialogs    = get_dialogs
```

Comment:
Does it matter that this now happens AFTER the `exit_pre_dialog` does all the `@values` checks?